x-pack/plugins/cases/public/components/header_page/title.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiBetaBadge, EuiBadge, EuiTitle } from '@elastic/eui';
+import styled from 'styled-components';
+
+import { DraggableArguments, BadgeOptions, TitleProp } from './types';
+
+const StyledEuiBetaBadge = styled(EuiBetaBadge)`
+  vertical-align: middle;
+`;
+
+StyledEuiBetaBadge.displayName = 'StyledEuiBetaBadge';
+
+const Badge = (styled(EuiBadge)`
+  letter-spacing: 0;
+` as unknown) as typeof EuiBadge;
+Badge.displayName = 'Badge';
+
+interface Props {
+  badgeOptions?: BadgeOptions;
+  title: TitleProp;
+  draggableArguments?: DraggableArguments;
+}
+
+const TitleComponent: React.FC<Props> = ({ draggableArguments, title, badgeOptions }) => (
+  <EuiTitle size="l">
+    <h1 data-test-subj="header-page-title">
+      {/* TODO: Enable Draggables here
```

Comment:
is this the only draggable you found? We definitely do not need this. It is only used on the Security network page. Safe to delete

Reply:
Yep! Removed 👍🏾

Reply:
nice